From patchwork Sun Nov 22 13:03:44 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Hugh Cole-Baker <sigmaris@gmail.com>
X-Patchwork-Id: 1404474
X-Patchwork-Delegate: ykai007@gmail.com
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=2a01:238:438b:c500:173d:9f52:ddab:ee01; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=P6CnMO7U;
	dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de
 [IPv6:2a01:238:438b:c500:173d:9f52:ddab:ee01])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits))
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4Cf9VN30jjz9sT6
	for <incoming@patchwork.ozlabs.org>; Mon, 23 Nov 2020 00:04:41 +1100 (AEDT)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
	by phobos.denx.de (Postfix) with ESMTP id CE775825A1;
	Sun, 22 Nov 2020 14:04:18 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="P6CnMO7U";
	dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id 74E088257F; Sun, 22 Nov 2020 14:04:13 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE,URIBL_BLOCKED
 autolearn=ham autolearn_force=no version=3.4.2
Received: from mail-wr1-x441.google.com (mail-wr1-x441.google.com
 [IPv6:2a00:1450:4864:20::441])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 898AE82564
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 14:04:09 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=sigmaris@gmail.com
Received: by mail-wr1-x441.google.com with SMTP id l1so15714256wrb.9
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 05:04:09 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:cc:subject:date:message-id:in-reply-to:references
 :mime-version:content-transfer-encoding;
 bh=6SYQmHVr0D0O3wWmxMfhj7ZKtMAcYMPC+fsr+GotKTs=;
 b=P6CnMO7Up+Z5K5CbVzKd4H/2QN8yjMOXmyDou0LODbNDm9Bk5yNIT7IPUTNgCi7OHU
 MCkWj1sA/Y0Vy7be+3ft1nvEIh1fY6E44ffw6tGcvGUf7rDXyQvyAZ6zSd8IHyqI44Xq
 dzs/KERwAULCzSHt9OQNGYDnv3bKJAfGhh5t+nfR8z8/9ssONR3q18wKQEJ1X5eFO/IK
 butM9Dq2sfJ0e15aYShs2Ku0k641pHX6ur5zSujD46MVhPApRhbHN9pbTAdmBmFb5cpk
 taUyiQmhmHxj8oclIibEwDHS/0ygI7OmaEfDZRoYa5K8U6dp7hqDb9iZHasNsyBYRHs+
 Iuyg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references:mime-version:content-transfer-encoding;
 bh=6SYQmHVr0D0O3wWmxMfhj7ZKtMAcYMPC+fsr+GotKTs=;
 b=mQwI4z7mnmNifc4yfgZj/vpgd1vQW9IOxjw2r3+sk+pRuJ6SjrjPsFmnfU8d1CfA5X
 /IgzqDbWRhYtjMPiXe0VbrS8HfI5Fqps/BFq2BjN+zrER/ZUKC0MSkv7XsYQmM3UoFgo
 ysyImd1T8hkM2fcsg0idMJ1AAY2Mtol2oKwHm+F5kW054bkSUo2mWpGRJE/RUTGMJYzP
 oGu6T7V/CLjZCuWNJkpzwzCY7BoyLnirZ+EH4mMGkxJgZ4NdZIl+DVP0kcbSgNLTDMhy
 JTF57D+dog/31smuIZ38Xtiee9OlTyoaTQzEN8+1zfHSevwQyUD48xyKcLFA2DVAYtYP
 X4wQ==
X-Gm-Message-State: AOAM533BH77QNc2JvGEEMb3O1vShAryk75TxbZD57eNvHYZ5XNi0l3Nd
 Hi9Vz0PAFdp4uFVQjchxkUeQHvOpOpzcpA==
X-Google-Smtp-Source: 
 ABdhPJzsTBCcXrYUm/5t74Gtg0n/vpUspyjdh6Ua796/vUxpSvKtaPDvdYxfFz+jaI/Ujmpb8WJFOw==
X-Received: by 2002:a5d:5183:: with SMTP id k3mr5107660wrv.62.1606050248908;
 Sun, 22 Nov 2020 05:04:08 -0800 (PST)
Received: from apple.sigmaris.info (ebrouter.sigmaris.info. [82.69.107.165])
 by smtp.gmail.com with ESMTPSA id i6sm10968566wma.42.2020.11.22.05.04.08
 (version=TLS1_2 cipher=ECDHE-ECDSA-AES128-GCM-SHA256 bits=128/128);
 Sun, 22 Nov 2020 05:04:08 -0800 (PST)
From: Hugh Cole-Baker <sigmaris@gmail.com>
To: U-Boot Mailing List <u-boot@lists.denx.de>
Cc: Hugh Cole-Baker <sigmaris@gmail.com>, Simon Glass <sjg@chromium.org>,
 Kever Yang <kever.yang@rock-chips.com>,
 Peter Robinson <pbrobinson@gmail.com>,
 Philipp Tomsich <philipp.tomsich@theobroma-systems.com>
Subject: [PATCH v2 1/3] rockchip: pinebook-pro: default to SPI bus 1 for
 SPI-flash
Date: Sun, 22 Nov 2020 13:03:44 +0000
Message-Id: <20201122130346.11621-2-sigmaris@gmail.com>
X-Mailer: git-send-email 2.24.3 (Apple Git-128)
In-Reply-To: <20201122130346.11621-1-sigmaris@gmail.com>
References: <20201122130346.11621-1-sigmaris@gmail.com>
MIME-Version: 1.0
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.3 at phobos.denx.de
X-Virus-Status: Clean

SPI flash on this machine is located on bus 1, default to using bus 1
for SPI flash and stop aliasing it to bus 0.

Signed-off-by: Hugh Cole-Baker <sigmaris@gmail.com>
Suggested-by: Simon Glass <sjg@chromium.org>
Fixes: c4cea2bb ("rockchip: Enable building a SPI ROM image on bob")
---

(no changes since v1)

 arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi | 4 ----
 configs/pinebook-pro-rk3399_defconfig        | 2 ++
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi b/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
index ded7db0aeff..ee3b98698e6 100644
--- a/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-pinebook-pro-u-boot.dtsi
@@ -7,10 +7,6 @@
 #include "rk3399-sdram-lpddr4-100.dtsi"
 
 / {
-	aliases {
-		spi0 = &spi1;
-	};
-
 	chosen {
 		u-boot,spl-boot-order = "same-as-spl", &sdhci, &spiflash, &sdmmc;
 	};
diff --git a/configs/pinebook-pro-rk3399_defconfig b/configs/pinebook-pro-rk3399_defconfig
index 1ed54ad0eda..ef00ad92c45 100644
--- a/configs/pinebook-pro-rk3399_defconfig
+++ b/configs/pinebook-pro-rk3399_defconfig
@@ -37,6 +37,7 @@ CONFIG_SPL_OF_CONTROL=y
 CONFIG_OF_SPL_REMOVE_PROPS="pinctrl-0 pinctrl-names clock-names interrupt-parent assigned-clocks assigned-clock-rates assigned-clock-parents"
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_SYS_RELOC_GD_ENV_ADDR=y
+CONFIG_SPL_DM_SEQ_ALIAS=y
 CONFIG_ROCKCHIP_GPIO=y
 CONFIG_SYS_I2C_ROCKCHIP=y
 CONFIG_DM_KEYBOARD=y
@@ -49,6 +50,7 @@ CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_SDMA=y
 CONFIG_MMC_SDHCI_ROCKCHIP=y
+CONFIG_SF_DEFAULT_BUS=1
 CONFIG_SF_DEFAULT_SPEED=20000000
 CONFIG_SPI_FLASH_GIGADEVICE=y
 CONFIG_SPI_FLASH_WINBOND=y

From patchwork Sun Nov 22 13:03:45 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Hugh Cole-Baker <sigmaris@gmail.com>
X-Patchwork-Id: 1404476
X-Patchwork-Delegate: ykai007@gmail.com
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=85.214.62.61; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=REQwZbU4;
	dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de [85.214.62.61])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256)
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4Cf9Vp2Yl9z9sSf
	for <incoming@patchwork.ozlabs.org>; Mon, 23 Nov 2020 00:05:05 +1100 (AEDT)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
	by phobos.denx.de (Postfix) with ESMTP id 99CC4825B4;
	Sun, 22 Nov 2020 14:04:20 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="REQwZbU4";
	dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id 0345B825A6; Sun, 22 Nov 2020 14:04:16 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE,URIBL_BLOCKED
 autolearn=ham autolearn_force=no version=3.4.2
Received: from mail-wr1-x443.google.com (mail-wr1-x443.google.com
 [IPv6:2a00:1450:4864:20::443])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 02CAF821A4
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 14:04:12 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=sigmaris@gmail.com
Received: by mail-wr1-x443.google.com with SMTP id j7so15775922wrp.3
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 05:04:12 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:cc:subject:date:message-id:in-reply-to:references
 :mime-version:content-transfer-encoding;
 bh=/Cjx25mfLwaytYlHbceqeQvGw33xUtrml0CUl/sig90=;
 b=REQwZbU43XaBkdSFJC5QRtUHq4gYwGxcC45P5486nZCm8jPwqzynI8LFxIAXAgdwJw
 0Bi8vA+7NuNZPwWqmpr96OlvJ8jOnCw07UBhhyKIr3BnUjXY2VCCnqQIGUt0+Zc5FVRl
 LGknOPa01vr6byuTe6OmEfcXbfW/w2RFpVLM7ICR31z8wIyiIW/W8lJu1etHg7hc+pEG
 z0c7dcYuwSaHyiyMr95cUVTsBd1CYAwBGDmKr88XOgPfvU7WGJSeVSkGp2ktPFEDcXxA
 E9GjcB+6aefFsz3ih5qO7sIGcb6z2UenEhkTvVnbqn6Or4x9SmK/kfTmku5NYuDfQoYZ
 eH6g==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references:mime-version:content-transfer-encoding;
 bh=/Cjx25mfLwaytYlHbceqeQvGw33xUtrml0CUl/sig90=;
 b=c21R1EknGJQ1BqTWS2Ua1SE2Tl/wfNHXA1okA7kIO/ktuWylvwLUocFaS3+TMubFzA
 dux0pHU9EemHTMuZchZZxyAP5Mw57UJjqQt3YNaAn5YyqhSLQ9kW32EccruhA97k0Vr1
 lUimK0Y4Sovm2CeedFfLKJL9VkxNM+ztzZqPQCLsejd8R4/e86PccNI/s2fEYte1PF33
 yMbcxDafcTGjBTl5q34iKluq/Hh7hC61sZ7UJIM/Bs73G0JdRDT4jH2HluNEKTkoDj8g
 1sL9TYanE4B3M5zvdDCTmp/pEFVa5xRvK9ZnsGC0hZ6ZkoiNW0PWA8Lo823Gd3bUPn2M
 EkKw==
X-Gm-Message-State: AOAM532pWtRzHv8t9BYLRQ1FOrka0x3QbsTIGLOM4it1BgaFt2AjMMyK
 WJshrm930J60FAT3/aTk+rwBX8wFJWnXrA==
X-Google-Smtp-Source: 
 ABdhPJxlXmDDU2QAEnJ1Ygx4v3gpz3fjzlmm3EMlmio1HHK/iH5XnwEw8detp4XdCi10r0mdAPKH0g==
X-Received: by 2002:adf:e788:: with SMTP id n8mr25250437wrm.84.1606050252459;
 Sun, 22 Nov 2020 05:04:12 -0800 (PST)
Received: from apple.sigmaris.info (ebrouter.sigmaris.info. [82.69.107.165])
 by smtp.gmail.com with ESMTPSA id i6sm10968566wma.42.2020.11.22.05.04.11
 (version=TLS1_2 cipher=ECDHE-ECDSA-AES128-GCM-SHA256 bits=128/128);
 Sun, 22 Nov 2020 05:04:11 -0800 (PST)
From: Hugh Cole-Baker <sigmaris@gmail.com>
To: U-Boot Mailing List <u-boot@lists.denx.de>
Cc: Hugh Cole-Baker <sigmaris@gmail.com>, Simon Glass <sjg@chromium.org>,
 Jagan Teki <jagan@amarulasolutions.com>,
 Kever Yang <kever.yang@rock-chips.com>, Levin Du <djw@t-chip.com.cn>,
 Philipp Tomsich <philipp.tomsich@theobroma-systems.com>,
 Suniel Mahesh <sunil@amarulasolutions.com>
Subject: [PATCH v2 2/3] rockchip: rk3399-roc-pc: default to SPI bus 1 for
 SPI-flash
Date: Sun, 22 Nov 2020 13:03:45 +0000
Message-Id: <20201122130346.11621-3-sigmaris@gmail.com>
X-Mailer: git-send-email 2.24.3 (Apple Git-128)
In-Reply-To: <20201122130346.11621-1-sigmaris@gmail.com>
References: <20201122130346.11621-1-sigmaris@gmail.com>
MIME-Version: 1.0
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.3 at phobos.denx.de
X-Virus-Status: Clean

SPI flash on this board is located on bus 1, default to using bus 1 for
SPI flash on both rk3399-roc-pc and -mezzanine, and stop aliasing it to
bus 0.

Signed-off-by: Hugh Cole-Baker <sigmaris@gmail.com>
Suggested-by: Simon Glass <sjg@chromium.org>
Fixes: c4cea2bb ("rockchip: Enable building a SPI ROM image on bob")
---

(no changes since v1)

 arch/arm/dts/rk3399-roc-pc-u-boot.dtsi    | 4 ----
 configs/roc-pc-mezzanine-rk3399_defconfig | 1 +
 configs/roc-pc-rk3399_defconfig           | 1 +
 3 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/rk3399-roc-pc-u-boot.dtsi b/arch/arm/dts/rk3399-roc-pc-u-boot.dtsi
index fc155e69036..e3c9364e359 100644
--- a/arch/arm/dts/rk3399-roc-pc-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-roc-pc-u-boot.dtsi
@@ -7,10 +7,6 @@
 #include "rk3399-sdram-lpddr4-100.dtsi"
 
 / {
-	aliases {
-		spi0 = &spi1;
-	};
-
 	chosen {
 		u-boot,spl-boot-order = "same-as-spl", &spi_flash, &sdhci, &sdmmc;
 	};
diff --git a/configs/roc-pc-mezzanine-rk3399_defconfig b/configs/roc-pc-mezzanine-rk3399_defconfig
index ae16f3558a3..8aa5a155180 100644
--- a/configs/roc-pc-mezzanine-rk3399_defconfig
+++ b/configs/roc-pc-mezzanine-rk3399_defconfig
@@ -42,6 +42,7 @@ CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_ROCKCHIP=y
+CONFIG_SF_DEFAULT_BUS=1
 CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_DM_ETH=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/roc-pc-rk3399_defconfig b/configs/roc-pc-rk3399_defconfig
index 774707b115b..927b57685d9 100644
--- a/configs/roc-pc-rk3399_defconfig
+++ b/configs/roc-pc-rk3399_defconfig
@@ -41,6 +41,7 @@ CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_ROCKCHIP=y
+CONFIG_SF_DEFAULT_BUS=1
 CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_DM_ETH=y
 CONFIG_ETH_DESIGNWARE=y

From patchwork Sun Nov 22 13:03:46 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Hugh Cole-Baker <sigmaris@gmail.com>
X-Patchwork-Id: 1404477
X-Patchwork-Delegate: ykai007@gmail.com
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=85.214.62.61; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=V6cH+Ab0;
	dkim-atps=neutral
Received: from phobos.denx.de (phobos.denx.de [85.214.62.61])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits))
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4Cf9W26DWNz9sSf
	for <incoming@patchwork.ozlabs.org>; Mon, 23 Nov 2020 00:05:18 +1100 (AEDT)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
	by phobos.denx.de (Postfix) with ESMTP id ED850825C8;
	Sun, 22 Nov 2020 14:04:29 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="V6cH+Ab0";
	dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id E1A45825AB; Sun, 22 Nov 2020 14:04:22 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.0 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,FREEMAIL_FROM,SPF_HELO_NONE,URIBL_BLOCKED
 autolearn=ham autolearn_force=no version=3.4.2
Received: from mail-wm1-x343.google.com (mail-wm1-x343.google.com
 [IPv6:2a00:1450:4864:20::343])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 5844D825AB
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 14:04:19 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=sigmaris@gmail.com
Received: by mail-wm1-x343.google.com with SMTP id 10so15317137wml.2
 for <u-boot@lists.denx.de>; Sun, 22 Nov 2020 05:04:19 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:cc:subject:date:message-id:in-reply-to:references
 :mime-version:content-transfer-encoding;
 bh=iIABfTCsAnamS6ADtBcJnJDUKIZeDRjQlCiKPeXJeqY=;
 b=V6cH+Ab04tdAtCynJa7Qo7eZ0vOL+fDJlPGfOsICfFxkuWCsAqo4YC1Lg+J1OoQvrF
 MVW8XT73Fltm2U5f+3rveS4CbcF+xbyf316XXSlZYzlV2MECQHM1hMMNHVYPBwW8Ny3W
 jMILQm5wpgigFT7L5V8Xiwfsoipi1OWBinnGWENTCgz/bpdQT4pPh61GYeGCtk3UPay+
 7I8RrljPl3ya1aEEoX3J1M3VrSoQrwmkL7uwcSTis52BODNDXLIthfIex7obwoKrfZ7F
 yrwOL+MGYuiiF5PEdGdruQmcxMWUwtMhS2RbElFRRjBlB0A16ofrAXpOOsSS+C18qXo+
 XIPg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references:mime-version:content-transfer-encoding;
 bh=iIABfTCsAnamS6ADtBcJnJDUKIZeDRjQlCiKPeXJeqY=;
 b=jzfmFxc0145sVD+QeBzKFADhk4C/yzyilURKZZaVznYqUzq4jpnOEUXOKrTDSml/kK
 dyHdpxX7AOj58BDsEbLUTmdhqIuQSJlh8wXPVCPMNvJ66hAwO9jZ/8Y7zYYyVzQ+RkzO
 653do0IvgG0Wb/cLLakIbtLIjL8rg+T70rI//qWGT4OrrVp/nI8U6IXnSAR1SxxCjGZQ
 W/3FudFguFZA76WoGX7ZiTveS4J3LkWY4hMRRVSXa10xZKIkgjIr6si5yl0WAhsJIzep
 NxN3WJiU0lTXarelq336e1kUvh+aU76hw43R0qcleFYXdunl/Z7ChfVelgpltmQqyLx7
 wgJg==
X-Gm-Message-State: AOAM531Du1DX9BT1NWQgESsUfd//f4j2LyAj2AOjJY5L/2VB5fARZ2cA
 x4CIUqCbM6DgJ9FYGJ629YAJ8eH9U7CC3g==
X-Google-Smtp-Source: 
 ABdhPJz9FBgRtp7SHTW2BICUUPHvdN1vEdLJP8V3sn7sYgqugP0WuiDc/DRtvRECan9YqreK37PSHA==
X-Received: by 2002:a1c:1dc1:: with SMTP id
 d184mr19004343wmd.169.1606050255890;
 Sun, 22 Nov 2020 05:04:15 -0800 (PST)
Received: from apple.sigmaris.info (ebrouter.sigmaris.info. [82.69.107.165])
 by smtp.gmail.com with ESMTPSA id i6sm10968566wma.42.2020.11.22.05.04.15
 (version=TLS1_2 cipher=ECDHE-ECDSA-AES128-GCM-SHA256 bits=128/128);
 Sun, 22 Nov 2020 05:04:15 -0800 (PST)
From: Hugh Cole-Baker <sigmaris@gmail.com>
To: U-Boot Mailing List <u-boot@lists.denx.de>
Cc: Hugh Cole-Baker <sigmaris@gmail.com>,
 Kever Yang <kever.yang@rock-chips.com>,
 Heiko Stuebner <heiko.stuebner@theobroma-systems.com>,
 Jagan Teki <jagan@amarulasolutions.com>,
 Klaus Goger <klaus.goger@theobroma-systems.com>,
 Peter Robinson <pbrobinson@gmail.com>,
 Philipp Tomsich <philipp.tomsich@theobroma-systems.com>,
 Simon Glass <sjg@chromium.org>
Subject: [PATCH v2 3/3] rockchip: puma-haikou: default to SPI bus 1 for
 SPI-flash
Date: Sun, 22 Nov 2020 13:03:46 +0000
Message-Id: <20201122130346.11621-4-sigmaris@gmail.com>
X-Mailer: git-send-email 2.24.3 (Apple Git-128)
In-Reply-To: <20201122130346.11621-1-sigmaris@gmail.com>
References: <20201122130346.11621-1-sigmaris@gmail.com>
MIME-Version: 1.0
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.102.3 at phobos.denx.de
X-Virus-Status: Clean

SPI flash on this machine is located on bus 1, default to using bus 1
for SPI flash and stop aliasing it to bus 0. Formerly the alias spi1
pointed to &spi5, use an alias spi5 for this instead.

Signed-off-by: Hugh Cole-Baker <sigmaris@gmail.com>
Suggested-by: Kever Yang <kever.yang@rock-chips.com>
---

Changes in v2:
New patch.

 arch/arm/dts/rk3399-puma-haikou-u-boot.dtsi | 3 +--
 configs/puma-rk3399_defconfig               | 2 ++
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/rk3399-puma-haikou-u-boot.dtsi b/arch/arm/dts/rk3399-puma-haikou-u-boot.dtsi
index 29846c4b001..e7a1aea3a56 100644
--- a/arch/arm/dts/rk3399-puma-haikou-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-puma-haikou-u-boot.dtsi
@@ -28,8 +28,7 @@
 	};
 
 	aliases {
-		spi0 = &spi1;
-		spi1 = &spi5;
+		spi5 = &spi5;
 	};
 
 	/*
diff --git a/configs/puma-rk3399_defconfig b/configs/puma-rk3399_defconfig
index 60178e3754c..0f15936c2d6 100644
--- a/configs/puma-rk3399_defconfig
+++ b/configs/puma-rk3399_defconfig
@@ -44,6 +44,7 @@ CONFIG_ENV_OVERWRITE=y
 CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_SYS_MMC_ENV_DEV=1
+CONFIG_SPL_DM_SEQ_ALIAS=y
 CONFIG_ROCKCHIP_GPIO=y
 CONFIG_SYS_I2C_ROCKCHIP=y
 CONFIG_MISC=y
@@ -53,6 +54,7 @@ CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_SDMA=y
 CONFIG_MMC_SDHCI_ROCKCHIP=y
+CONFIG_SF_DEFAULT_BUS=1
 CONFIG_SF_DEFAULT_SPEED=20000000
 CONFIG_SPI_FLASH_GIGADEVICE=y
 CONFIG_SPI_FLASH_WINBOND=y
